src/manifold/src/manifold.cpp

```diff
@@ -845,4 +848,141 @@ Manifold Manifold::Hull() const { return Hull(GetMesh().vertPos); }
 Manifold Manifold::Hull(const std::vector<Manifold>& manifolds) {
   return Compose(manifolds).Hull();
 }
+
+// TODO: Handle Joggling in the Fracture Function Directly?
+/**
+ * Compute the voronoi fracturing of this Manifold into convex chunks.
+ *
+ * @param pts A vector of points over which to fracture the manifold.
```

Comment:
Can we have some detail on what these inputs do and the purpose of this function?